From 67278876595ece02622d7dcb4daa6c4264f87059 Mon Sep 17 00:00:00 2001
From: Ciprian Costea <ciprianmarian.costea@nxp.com>
Date: Wed, 13 Apr 2022 09:52:54 +0300
Subject: [PATCH 094/103] s32cc: Remove double 'ft_fixup_cpu' call

This call should have been removed as part of
'ft_system_setup' addition, but it was omitted.

Issue: ALB-8588
Upstream-Status: Pending 

Signed-off-by: Ciprian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/mach-s32/s32-cc/fdt.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-s32/s32-cc/fdt.c b/arch/arm/mach-s32/s32-cc/fdt.c
index 5d8bf1c868..368d7307e4 100644
--- a/arch/arm/mach-s32/s32-cc/fdt.c
+++ b/arch/arm/mach-s32/s32-cc/fdt.c
@@ -715,7 +715,6 @@ static void ft_fixup_serdes(void *blob)
 
 void ft_cpu_setup(void *blob, bd_t *bd)
 {
-	ft_fixup_cpu(blob);
 	ft_fixup_memory(blob, bd);
 	ft_fixup_atf(gd->fdt_blob, blob);
 #ifdef CONFIG_PCIE_S32GEN1
-- 
2.25.1

